BuildScript.Nuke/Build.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using Nuke.Common;
+using Nuke.Common.Execution;
+using Nuke.Common.Git;
+using Nuke.Common.IO;
+using Nuke.Common.ProjectModel;
+
+[CheckBuildProjectConfigurations]
+internal partial class Build : NukeBuild
+{
+  [GitRepository] private readonly GitRepository GitRepository;
+
+  [Solution] private readonly Solution Solution;
+
+  private DirectoryHelper DirectoryHelper { get; } = new(RootDirectory);
+  private AssemblyMetadata AssemblyMetadata { get; set; }
+  private VersionHelper VersionHelper { get; set; }
+
+  public IEnumerable<ReleaseProjectMetadata> ReleaseProjectFiles { get; set; } = new List<ReleaseProjectMetadata>();
+  public Dictionary<AbsolutePath, string> UnitTestProjectFiles { get; set; } = new();
```

Comment:
do these properties need to be public?

```suggestion
  private IEnumerable<ReleaseProjectMetadata> ReleaseProjectFiles { get; set; } = new List<ReleaseProjectMetadata>();
  private Dictionary<AbsolutePath, string> UnitTestProjectFiles { get; set; } = new();
```